modules/devices/e2k/processor.c

```diff
@@ -238,6 +238,32 @@ gchar *caches_summary(GSList * processors)
     return ret;
 }
 
+gchar *processor_get_full_name(gchar *model_name)
+{
+    gchar *full_name;
+
+    if(g_strcmp0(model_name ,"E2S") == 0)
+        full_name = g_strdup("Elbrus-4C");
```

Comment:
```suggestion
        return "Elbrus-4C";
```

(Same for all others here: there's no need to allocate a new string.)